stock_picking_auto_create_lot/tests/test_stock_picking_auto_create_lot.py

```diff
@@ -90,12 +91,10 @@ def test_auto_create_transfer_lot(self):
         moves = self.picking.move_lines.filtered(
             lambda m: m.product_id == self.product_serial_not_auto
         )
-
         # Assign manual serials
-        for line in moves.mapped("move_line_ids"):
-            line.lot_id = self.lot_obj.create(line._prepare_auto_lot_values())
+        self._assign_manual_serials(moves)
 
-        self.picking.button_validate()
+        self.picking.with_context(skip_immediate=True).button_validate()
```

Comment:
I guess we don't need this adjustment.
```suggestion
        self.picking.button_validate()
```